### ISSUE TITLE
Viewlets should have appropriate heading level tag

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#51760]
**Environment Details:**
VSCode Version : 1.24.0
OS Version : Win10
**Additional Details:**
ScreenReader,NVDA[2018.2],JAWS[18.0.5038],MAS1.3.1
**Repro Steps:**
1. Launch VS Code.
2. Navigate to Activity Bar and select "View Extensions "(Cntrl+Shift+X) button.
3. Navigate to complementary landmark -"View Extensions " section and read the content "View Extensions" using basic navigation.

**Actual:**
Screenreader does not read the content "View Extensions" as appopriate heading which is so.

Same issue has been observed with JAWS.

**Expected:**
Screenreader should read- "View Extensions" as heading with appropriate heading level.

**Recommendations:**
Add appropriate native heading level tag to the content "View Extensions".
or,
Refer below link which is repository of bug fixes code snippets:
[https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx](url)

User Impact:
The screenreader users will not be able to know the appropriate headings present on the screen.

MAS Reference:
[https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}](url)
Does this issue occur when all extensions are disabled?: Yes
**Attachment for Reference:** 
![accessibility a11y_vscode_viewextensions_screenreader2](https://user-images.githubusercontent.com/39204909/41456454-1d48d0e4-709e-11e8-9a27-8373dff90b05.png)
[[Accessibility]A11y_VSCode_ViewExtensions_ScreenReader3.pptx](https://github.com/Microsoft/vscode/files/2105214/Accessibility.A11y_VSCode_ViewExtensions_ScreenReader3.pptx)

**Comment by isidorn** [comment]
This should be done for all viewlets, not just extensions.
@JacksonKearl @ramya-rao-a do you plan to look into this or should I forward to someone else?
In case you do I guess this should be done somewhere in the `CompositePart` https://github.com/Microsoft/vscode/blob/9ab41a4464e62739fa8850d2aa312de2f72eb20d/src/vs/workbench/browser/parts/compositePart.ts#L409

**Comment by JacksonKearl** [comment]
I will look into this. Changing the title to be a heading rather than span doesn't seem to do anything. Will investigate further.

**Comment by isidorn** [comment]
@JacksonKearl thanks for looking into this.

**Comment by ramya-rao-a** [comment]
Note to verifier:
Check all viewlets, not just the one for Extensions

**Comment by JacksonKearl** [comment]
Also extension contributed viewlets. (GitLens, Azure App Service, etc.)

**Comment by JacksonKearl** [comment]
To view headings with NVDA, press `Capslock+Space` to toggle NVDA mode, then `h` and `Shift+h` to go forward and backward in headings.
